### PR TITLE
Fixed for updated ev3dev api

### PR DESCRIPTION
**Pull request by jermon**
ev3dev have changed to using address in the file API.

### CODE REVIEW

File: Motor/motors.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func findFolder(port OutPort) string {
 		}
 	}
 
-	log.Fatal("No motor is connected to port %s", port)
+	log.Fatal("No motor is connected to port ", port )
```
Comment (ldmberman) [inline Motor/motors.go]: Have you removed the %s by mistake?

Comment (jermon) [inline Motor/motors.go]: No. log.Fatal acts like Println not printf. 
It actually prints "No motor is connected to port %sA"
My correction prints "No motor is connected to port A"

Comment (ldmberman) [inline Motor/motors.go]: Indeed, thanks.

**Comment by ldmberman** [comment]
Thank you for the update!

**Pull request merged**